### ISSUE TITLE
0.31.7 PadiHash broken on Nvidia (AMD is ok)

### ISSUE THREAD

**Issue by UselessGuru** [andru-kun/wildrig-multi#136]
`wildrig.exe --algo glt-padihash --watchdog --api-port 4006 --url stratum+tcp://padihash.eu.mine.zpool.ca:8670 --user *******************--pass ****************,c=BTC --multiple-instance --opencl-platforms 0 --opencl-devices 0`

```
[09:34:24] initializing GPUs...
[09:34:24] PTX ISA: 7.7
[09:34:24] GPU #0: NVIDIA GeForce GTX 1660 SUPER [busID: 11] [arch: sm75]
[09:34:24] threads: 1, intensity: 21, worksize: 256/1024, cu: 44, mem: 6143Mb
[09:34:25] use pool padihash.eu.mine.zpool.ca:8670 198.50.168.213
[09:34:25] Stratum set raw difficulty to 4.0000
[09:34:25] new job from padihash.eu.mine.zpool.ca:8670 diff 67.11M/46.46G
[09:34:25] block: 2,128,309     job target: 0x0000003fffc00000
[09:34:25] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
[09:34:26] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
[09:34:27] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
[09:34:28] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
[09:34:29] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
[09:34:30] Error CL_INVALID_ARG_SIZE when calling clSetKernelArg for kernel 'bmw_512_64_final', argument 3
```

**Comment by UselessGuru** [comment]
Still broken in 0.3.18

**Comment by andru-kun** [comment]
can you try it with 0.32.0?

**Comment by UselessGuru** [comment]
Fixed with 0.32.0. Thank you!
